spanner/setup.py

```diff
@@ -31,7 +31,7 @@
 dependencies = [
     'google-cloud-core<0.29dev,>=0.28.0',
     'google-api-core[grpc, grpcio-gcp]<2.0.0dev,>=1.4.0',
-    'grpc-google-iam-v1<0.12dev,>=0.11.4',
+    'grpc-google-iam-v1<0.12dev,>=0.11.4'
```

Comment:
Always prefer trailing comma in lists like this, even where it isn't needed:  it eases future maintenance.